install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -113,6 +113,10 @@ spec:
 {{- if .Values.enableProtocolSniffing }}
           - name: PILOT_ENABLE_PROTOCOL_SNIFFING
             value: "{{ .Values.enableProtocolSniffing }}"
+{{- end }}
+{{- if .Values.enableProxyClientOriginalAddress }}
+          - name: PILOT_ENABLE_PROXY_CLIENT_ORIGINAL_ADDRESS
+            value: "1"
```

Comment:
enableProxyClientOriginalAddress

Reply:
?

Reply:
I mean the value should be from `.Values.enableProxyClientOriginalAddress`, not a fixed val

Reply:
Should be bool type as enableProtocolSniffing

Reply:
@mmerrill3 can you provide a fix for that?